RELEASE/scripts/autoscend/iotms/mr2020.ash

```diff
@@ -259,4 +259,65 @@ boolean auto_getGuzzlrCocktailSet() {
 		}
 	}
 	return false;
-}
\ No newline at end of file
+}
+
+boolean auto_latheHardwood(item toLathe)
+{
+	// can't lathe if lathe is out of standard (or otherwise unusable)
+	if(!auto_is_valid($item[SpinMaster&trade; lathe]))
+		return false;
+
+	// can't lathe... without a lathe
+	if(item_amount($item[SpinMaster&trade; lathe]) < 1)
+		return false;
+
+	// can't lathe without hardwood
+	if(item_amount($item[flimsy hardwood scraps]) < 1)
+		return false;
+
+	// can't lathe things that aren't made of hardwood
+	if(!($items[
+		beechwood blowgun,
+		birch battery,
+		ebony epee,
+		maple magnet,
+		weeping willow wand,
+	] contains toLathe))
+		return false;
+
+	return buy($coinmaster[Your SpinMaster&trade; lathe], 1, toLathe);
+}
+
+boolean auto_latheAppropriateWeapon()
+{
+	switch(my_class())
+	{
+		// autoscend likes Plumber to go for moxie, so let's make sure it
+		// does even if another stat is ahead at the start of the day.
+		case $class[Plumber]:
+			return auto_latheHardwood($item[beechwood blowgun]);
```

Comment:
```suggestion
			if (!possessEquipment($item[beechwood blowgun]))
			{
				return auto_latheHardwood($item[beechwood blowgun]);
			}
			else
			{
				return false;
			}
```
Otherwise plumber will end up with one for each day.

Reply:
Technically it won't since they melt at the end of the day and you can't go casual plumber, but this still seems like a thing to fix regardless, so fix I shall.

Reply:
> Technically it won't since they melt at the end of the day and you can't go casual plumber, but this still seems like a thing to fix regardless, so fix I shall.

LoL that shows you how much I've used it since it was released.